package.json

```diff
@@ -50,7 +38,19 @@
       "last 1 safari version"
     ]
   },
+  "dependencies": {
+    "@emmetio/codemirror-plugin": "^1.2.3",
+    "codemirror": "^5.60.0",
+    "node-sass": "^6.0.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
```

Comment:
Those things that we don't need in prod, should be moved to `devDependencies`. We don't want to have those modules in our chunk, so we could skip them, that's why I rewrote the code like this.